fem/tmop.cpp

```diff
@@ -3151,56 +3168,47 @@ void TMOP_Integrator::AssembleElemVecSurfFit(const FiniteElement &el_x,
    const FiniteElement &el_s = *surf_fit_gf->FESpace()->GetFE(el_id);
```

Comment:
Function params `ir_quad` and `weights` can be removed, and the same for the `GradSurfFit` function.